CONTRIBUTION.md

```diff
@@ -53,24 +53,24 @@ Make sure you have the following prerequisites installed on your operating syste
 
 Follow the following instructions to start contributing.
 
-**1.** Fork [this](https://github.com/kubesimplify/website) repository.
+**1.** Fork [website](https://github.com/kubesimplify/website/fork) repository to your own repositories.
 
-**2.** Clone your forked copy of the project.
+**2.**  Clone your forked copy of the project by clicking on the code button copy the HTTPS link.
```

Comment:
Missing a punctuation, "by clicking on the code button , copy the HTTPS link"

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix